lib/playground.dart

```diff
@@ -202,13 +225,11 @@ class Playground {
     DSplash splash = new DSplash(querySelector('div.splash'));
     splash.hide();
 
-    // TODO: I really want to be able to have paths without hashes in them.
-    // So, dartpad.foo/123456af rather then dartpad.foo/#123456af.
-    Router router = new Router(useFragment: true);
+    Router router = new Router();
     router.root.addRoute(
         name: 'home', defaultRoute: true, path: '/', enter: showHome);
     router.root.addRoute(
-        name: 'gist', path: ':gist', enter: showGist);
+        name: 'gist', path: '/:gist/', enter: showGist);
```

Comment:
@justinfagnani, I'm having trouble grokking routing. What I'm trying to do is get `/aef4325b` to route to `showGist()` with the gist id as a parameter. None of the many variations I've tried here has produced much success - generally it just routes to the home route. I know you've done a lot of work with both routing packages - any ideas?

I was able to get the routing to work with the fragment identifier (`#`), but would much prefer a url design that didn't have a hash.